config/locales/en.yml

```diff
@@ -305,6 +305,8 @@ en:
       lettype:
         general_needs_mismatch: Lettings type must be a general needs type because you selected general needs when uploading the file
         supported_housing_mismatch: Lettings type must be a supported housing type because you selected supported housing when uploading the file
+      location:
+        incomplete: "This location is incomplete"
```

Comment:
Has this copy been approved? We normally say something like "Select another location or update this one" in this case to make it clear to the user what to do

Reply:
I'm not sure if this is actually possible to trigger in the UI, I think it's more for import logging, but I'll ask in the ticket. I'll also add your suggestion